### PR TITLE
[#8458] [#8448] developed popover, share accounts and signing phrase …

### LINKED ISSUE #8448: [Multi-Account] Show signing phrase
### Description

*Type*: Feature

*Summary*: User should see [signing phrase modal](https://www.figma.com/file/XUehMnhyD1FGcWzvGz6SXqvh/Mobile-wallet?node-id=2474%3A170) in account explorer when opening it for the first time.

#### Expected behavior

- Signing phrase modal is displayed on a user's first visit to the wallet tab.
- If the user clicks, `Remind me later`, it appears again on subsequent visit to the tab.
- If the user clicks, `OK, got it`, it is not displayed again.

#### Actual behavior

Screen is not built yet.

### LINKED ISSUE #8458: [Multi-Account] Share wallet with pop-up design
### Description
_Type_: Feature

_Summary_: This issue is about implementing the appropriate sharing UI for this icon:

image

#### Expected behavior
Clicking share button launches popover with QR code and wallet field to copy.

[Figma link](https://www.figma.com/file/XUehMnhyD1FGcWzvGz6SXqvh/Mobile-wallet?node-id=2556%3A31102)

#### Actual behavior
Clicking share button launches native options:

### PR DESCRIPTION
**Pull request by flexsurfer**
fixes #8458
fixes #8448

### CODE REVIEW

File: src/status_im/ui/screens/wallet/navigation.cljs [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
-            [status-im.ui.screens.navigation :as navigation]))
+            [status-im.ui.screens.navigation :as navigation]
+            [status-im.ui.screens.wallet.signing-phrase.views :as signing-phrase]))
 
 (def transaction-send-default
   {:method constants/web3-send-transaction
    :symbol :ETH})
 
-(defmethod navigation/preload-data! :wallet-request-transaction
+(defmethod navigation/preload-data! :wallet-stack
```
Comment (jeluard) [inline src/status_im/ui/screens/wallet/navigation.cljs:10]: Still use that? No params passing?

**Review by jeluard — APPROVED** [review]

**Review by dmitryn — APPROVED** [review]

File: src/status_im/ui/components/qr_code_viewer/views.cljs [hunk 2]
```diff
@@ -11,23 +11,26 @@
...
      (when show-tribute-to-talk-warning?
        [react/view {:style {:margin-horizontal 16}}
         [tribute-to-talk/enabled-note]])
      (when width
-       (let [size (int (min width styles/qr-code-max-width))]
-         [react/view {:style               (styles/qr-code-container size)
-                      :accessibility-label :qr-code-image}
-          [qr-code {:value value
-                    :size  size}]]))
+       [qr-code-view (int (min width styles/qr-code-max-width)) value])
```
Comment (vitvly) [inline src/status_im/ui/components/qr_code_viewer/views.cljs:33]: Why is `int` conversion necessary here?

**Review by vitvly — APPROVED** [review]

File: src/status_im/ui/screens/wallet/navigation.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im/ui/screens/wallet/navigation.cljs:10]: yes, we want to show "signing phrase" when open wallet tab each time

File: src/status_im/ui/components/qr_code_viewer/views.cljs (hunk 2 above)
Comment (flexsurfer) [reply src/status_im/ui/components/qr_code_viewer/views.cljs:33]: just moved the code. have no idea :(

**Comment by asemiankevich** [comment]
@flexsurfer  is there is something we could improve in this branch? Thank you!

1. `This is your signing phrase` should be in bold

image

2. Share wallet code popup is glued to the bottom on Android
image


3. Poor performance of this Share wallet code on Android (blank screen is shown for a second)
https://drive.google.com/open?id=1yOLGMX9kGIUI2rIU8UP68IX_DGq778ag

4. `Send a transaction request` is multiline

image

5. Wallet Send transaction request screen remains active on the background when request is sent.

Steps:
1. open wallet
2. tap share button
3. tap Send transaction request
4. select a contact  -> amount -> send request
5. tap wallet button

https://drive.google.com/open?id=1UoA4sLsc0EdDrbi3gMs8h-BP_BkokB2F

**Comment by Serhy** [comment]
@flexsurfer please mind e2e fixes commit before force-push

**Comment by flexsurfer** [comment]
@asemiankevich fixed all except 3

**Pull request merged**